Jenkinsfile

```diff
@@ -9,7 +9,7 @@ pipeline {
   stages {
     stage('Test 2.2') {
```

Comment:
We should run these stages in parallel. I can add a commit to this PR showing how.